src/monitor/node_active_protocol.c

```diff
@@ -2389,17 +2388,11 @@ synchronous_standby_names(PG_FUNCTION_ARGS)
 		PG_RETURN_TEXT_P(cstring_to_text(""));
 	}
 
-	/* when we have more than one node, fetch the primary */
-	AutoFailoverNode *primaryNode = GetPrimaryNodeInGroup(formationId, groupId);
-
-	if (primaryNode == NULL)
-	{
-		ereport(ERROR,
-				(errmsg("Couldn't find the primary node in formation \"%s\", "
-						"group %d", formationId, groupId)));
-	}
+	AutoFailoverNode *primaryNode =
+		GetPrimaryNodeInGroup(formationId, groupId);
 
-	List *standbyNodesGroupList = AutoFailoverOtherNodesList(primaryNode);
+	List *standbyNodesGroupList =
+		AutoFailoverOtherNodesList(primaryNode);
```

Comment:
This looks like the only change is the indentation. I'd rather not review indent-only changes in a bug fix PR, can we get rid of this “noise”?